### PR TITLE
GL-1114 copied over skylab atac pipelines to warp

### PR DESCRIPTION
**Pull request by nikellepetrillo**

### CODE REVIEW

File: pipelines/skylab/scATAC/ATAC.options.json [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{
+  "write_to_cache": true,
+  "read_from_cache": true,
+  "monitoring_script": "gs://broad-biccn-dev-test-data/monitoring.sh",
```
Comment (kbergin) [inline pipelines/skylab/scATAC/ATAC.options.json:4]: Should this be updated to the gotc monitoring script that @jessicaway was talking about in slack this morning?

**Comment by kbergin** [comment]
@nikellepetrillo - is there a corresponding test to pull over with this pipeline?

**Comment by kbergin** [comment]
And docker images?

**Comment by nikellepetrillo** [comment]
@kbergin sorry! i actually copied over the dockers in this PR https://github.com/broadinstitute/warp/pull/24

**Comment by nikellepetrillo** [comment]
@kbergin and yes good point, i forgot to look for tests!

**Comment by nikellepetrillo** [comment]
closing this as the final work (copying over the tests) is being done here: https://github.com/broadinstitute/warp/pull/38

**Pull request closed without merging**